roles/define-paths/tasks/main.yml

```diff
@@ -43,17 +42,12 @@
 # Ex. /var/www/html/cardinald7.02dev/scripts
 - name: Set scripts_dir variable
   set_fact:
-    scripts_dir: "/var/www/html/{{ stack }}.02{% if acsf_environment=='' %}live{% elif acsf_environment=='test'%}test{% elif acsf_environment=='dev'%}dev{% endif %}/scripts"
+    scripts_dir: "/var/www/html/{{ stack }}.0{{ stack_id }}{% if acsf_environment=='' %}live{% elif acsf_environment=='test'%}test{% elif acsf_environment=='dev'%}dev{% endif %}/scripts"
```

Comment:
Why don't we define the whole stack id. It seems silly to assume 0 here.

Reply:
Because we use `stack_id` [here](https://github.com/SU-SWS/ansible-playbooks/blob/SITES-843-people-leland-stack/roles/setup-site/tasks/main.yml#L65) also.

Reply:
I guess if we get over 10 stacks and still need these scripts we are in trouble anyway.